### DISCUSSION TITLE
Grounded tag

### DISCUSSION THREAD

**Discussion by ghost** [janstehno/cotw-companion#12] [Ideas]
Just a small idea for the animals that need to be shot while in flight to get the trophy score. Potentially there could be some sort of label or symbol for them on their screen so we know the requirements and avoid the grounded penalty...

**Comment by janstehno** [comment]
Great idea! Like written in the discussion #9 I will try to find some way to show this "mini-information" in the app. It won't be a priority however.

**Reply by ghost** [reply]
No worries. Anything to do my part with an already fantastic app and help the wider community.

**Comment by janstehno** [comment]
Added in 1.6.6
